### DISCUSSION TITLE
Duda al publicar

### DISCUSSION THREAD

**Discussion by oscardanielM** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#552] [Q&A] [answered]
Hola

Ha alguien le ha salido este error al querer compilar en github pages

**Request failed with status code 502 github**

😥😥😥😥😥😥😥😥 solo falta eso

**Comment by codingcodax** [comment]
Ya lo solucionaste no?

**Reply by oscardanielM** [reply]
no aun sigo con el problema

**Comment by gersondlarosa** [comment]
Aun tiene duda

**Reply by oscardanielM** [reply]
yes, aun je

**Comment by oscardanielM** [comment, marked as answer]
YA resolví el error
